src/modules/70-pipeline/components/ArtifactsSelection/ArtifactInterface.ts

```diff
@@ -75,9 +77,10 @@ export interface ImagePathTypes {
   registryHostname?: string
   region?: any
   repositoryPort?: number | string
-  repository?: string
+  repository?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232297)
```suggestion
  repository?: unknown
```